src/main/java/graphql/schema/GraphqlDirectivesContainerTypeBuilder.java

```diff
@@ -22,6 +21,33 @@ public B replaceAppliedDirectives(List<GraphQLAppliedDirective> directives) {
         return (B) this;
     }
 
+    public B addAppliedDirectives(GraphQLAppliedDirective... directives) {
+        assertNotNull(directives, () -> "directives can't be null");
+        for (GraphQLAppliedDirective directive : directives) {
+            addAppliedDirective(directive);
+        }
+        return (B) this;
+    }
```

Comment:
This no longer clears existing directives

Reply:
Hmmmmmmmm or maybe not

```
    public B withDirectives(GraphQLDirective... directives) {
        assertNotNull(directives, () -> "directives can't be null");
        this.directives.clear();
        for (GraphQLDirective directive : directives) {
            withDirective(directive);
        }
        return (B) this;
    }
```

from another class does the same thing

copy and paste copy and paste bug??

Reply:
I can see other "withXXX plural" and they dont do the clear behavior

```
        public Builder withSchemaDirectives(GraphQLDirective... directives) {
            for (GraphQLDirective directive : directives) {
                withSchemaDirective(directive);
            }
            return this;
        }
```

Reply:
That's right we have a combo of a copy paste of possibly a bug and then a typo that led to unusual behaviour

My preference is to change `withXXX` rather than introducing new `addXXX` methods, because most people are used to builders having "with", but wasn't sure how cautious to be

Reply:
> y preference is to change withXXX rather than introducing new addXXX methods,
I concur.  Lets do that